lib/zold/node/farm.rb

```diff
@@ -165,11 +165,19 @@ def save(threads, list = [])
     def load
       @mutex.synchronize do
         if File.exist?(@cache)
-          AtomicFile.new(@cache).read.split(/\n/).map { |t| Score.parse(t) }
+          AtomicFile.new(@cache).read.split(/\n/).map { |t| parse_score_line(t)[0] }
         else
           []
         end
       end
     end
+
+    def parse_score_line(line)
+      score = []
```

Comment:
@emilianodellacasa possibly, return `0` for a non-existing value. I don't know if it's possible.